ROADMAP.md

```diff
@@ -0,0 +1,41 @@
+# `sget` Roadmap
+
+### Today
+
+`sget` fetches blobs from OCI registries, using Docker API registry auth, checking content digests to ensure integrity, and verifying any `cosign` signatures that are found in the registry.
+
+`sget` currently has no functionality to fetch arbitrary URLs.
+
+### Phase 0
+
+Add basic URL-fetching functionality to `sget`, with an optional ability to check content digests to ensure integrity.
```

Comment:
Does this mean that `--digest` is optional, and thus `sget` can download HTTP content with no checks whatsoever (akin to `curl`)?

I hadn't said this outloud, but the more I think about it, I was thinking this wouldn't be possible. I.e., for now, if you're fetching an arbitrary HTTP URL, there's no way for `sget` to succeed without `--digest`. And that once we add support for sig verification for HTTP content, you could use that in place of `--digest`. In other words, for HTTP, we would never not check a thing.

But I'm not sure! WDYT?

Reply:
Yeah, that was my thinking when I wrote that, because when we have later phases we want `--digest` to be optional anyway. Butt I guess we can make it required until then at least.

I think if we end up allowing some combination of flags/states that leads `sget` to be equivalent to `curl`, we should either fail (since we're not providing value) or at least warn loudly that what you're doing isn't safe at all.

Happy to remove the "optional" here, wdyt?

Reply:
+1 to requiring `--digest` until we have some support for social proof and/or maintainer policies